synaptic_reconstruction/imod/export.py

```diff
@@ -96,6 +135,9 @@ def export_segmentation(imod_path, mrc_path, object_id=None, output_path=None, r
         id_str = "" if object_id is None else f"for object {object_id}"
         raise RuntimeError(f"Segmentation extracted from {imod_path} {id_str} is empty.")
 
+    if depth_interpolation is not None:
+        segmentation = maximum_filter1d(segmentation, size=depth_interpolation, axis=0, mode="constant")
```

Comment:
This works very well to close the gaps in the masks exported from IMOD, at least for compartment masks. `depth_interpolation=10` seems to be a good setting since they tend to annotate every 10th slice.